code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -977,13 +1010,34 @@
 			. += aux
 		return
 
-
+	var/draw_color
 	if(should_draw_greyscale)
-		var/draw_color = mutation_color || species_color || (skin_tone && skintone2hex(skin_tone))
-		if(draw_color)
-			limb.color = "[draw_color]"
-			if(aux_zone)
-				aux.color = "[draw_color]"
+		draw_color = mutation_color || species_color || (skin_tone && skintone2hex(skin_tone))
+
+	if(is_husked)
```

Comment:
Why not check if the limb's owner is husked directly instead of needing a new variable for it? You can't de-husk individual limbs anyway.

Reply:
IIRC the reason for this was so that surgically removed/gibbed husk limbs retain the husk coloration

Reply:
> IIRC the reason for this was so that surgically removed/gibbed husk limbs retain the husk coloration

Attaching it to any other body will revert it back to normal anyway, retaining the husk coloration after dismemberment implies that it doesn't which would be confusing.

Reply:
only worthwhile if we port tg's bodyparty and organ refactor
and as of yet i don't see a benefit